src/Html2Text.php

```diff
@@ -396,6 +397,16 @@ protected function converter(&$text)
         }
     }
 
+    /**
+     * show/hide alt text for images
+     */
+
+    protected function convertImages(){
```

Comment:
Poor method naming. This does not actually convert images.
I'd suggest the same approach as `convertBlockquotes` and `covnertPre`where they take the  value and actually perform the conversion. This will need to be done immediately _after_ the current `preg_replace` to ensure b/c.